packages/xs-extra/xapi-storage.master/opam

```diff
@@ -1,25 +1,23 @@
 opam-version: "1.2"
+name: "xapi-storage"
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://xapi-project.github.io/"
 bug-reports: "https://github.com/xapi-project/xapi-storage/issues"
 dev-repo: "git://github.com/xapi-project/xapi-storage.git"
-build: [
-  [make]
-]
-install: [
-  [make "install" "PREFIX=%{prefix}%"]
-]
-remove: [
-  [make "uninstall" "PREFIX=%{prefix}%"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
+  "jbuilder" {build}
+  "ounit" {test}
+  "cow"
   "rpc"
-  "re"
+  "xmlm"
   "cmdliner"
 ]
+depopts: [
+  "caml2html"
+]
 depexts: [
-  [ ["debian"] ["python"] ]
```

Comment:
was this dropped because it was not tested, or is there something broken on debian about it?

Reply:
I changed it to centos because this is what we're mostly interested in since xenserver is based on it. I also remember having a discussion on this with @mseri a few days ago.

Reply:
Well I have python installed anyway on my Ubuntu dev machine, so in this case probably doesn't matter, for other packages we should probably try to keep the debian/ubuntu entries working since that is what our development environment is based on (`dev-vm` etc).

Reply:
I think I also need to add `camlmix` to the repo because `caml2html` depends on it.

Reply:
Ok I restored the original distros.